builder_data.yaml

```diff
@@ -50,10 +50,10 @@ apt_repo_command: |
   keychain -k mine;
 yum_repo_command: |
   for repodir in $(find "__REPO_PATH__" -mindepth 2 -name "*.rpm" | xargs -I {} dirname {} | uniq) ; do
-    [ -d "${repodir}" ] || continue;
-    echo "Generating repodata for ${repodir}..."
-    sudo createrepo --checksum=sha --database --update "${repodir}";
-    echo "Finished generating repodata for ${repodir}..."
+    [ -d "${repodir}" ] || continue &&
+    echo "Generating and signing repodata for ${repodir} with __GPG_KEY__..." &&
+    sudo createrepo --checksum=sha --database --update "${repodir}" &&
+    gpg --no-tty --use-agent --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml"
```

Comment:
there are also going to be permission issues here :( I haven't decided on the best way to get around this. 

```
melissa@weth:/opt/repository/yum/el/7/PC1/x86_64/repodata$ gpg --no-tty --use-agent --armor --detach-sign -u EF8D349F repomd.xml
gpg: can't create `repomd.xml.asc': Permission denied
gpg: signing failed: file create error
melissa@weth:/opt/repository/yum/el/7/PC1/x86_64/repodata$ sudo gpg --no-tty --use-agent --armor --detach-sign -u EF8D349F repomd.xml
gpg: WARNING: unsafe ownership on configuration file `/home/melissa/.gnupg/gpg.conf'
gpg: gpg-agent is not available in this session
gpg: Sorry, no terminal at all requested - can't get input
```